src/Controls/tests/BindingSourceGen.UnitTests/BindingRepresentationGenTests.cs

```diff
@@ -66,6 +76,12 @@ public void GenerateBindingWithLongerPath()
                 true
             );
         Assert.Equivalent(expectedBinding, actualBinding);
-    }
+        
+        Assert.Equal(expectedBinding.Path.Length, actualBinding.Path.Length); //TODO: Change arrays to custom collections implementing IEquatable
 
+        for (int i = 0; i < expectedBinding.Path.Length; i++)
+        {
+            Assert.Equal(expectedBinding.Path[i], actualBinding.Path[i]);
+        }
```

Comment:
Isn't there an `Assert.SequenceEqual` method or something like that?

Reply:
There is. However it did not show me which element of the array causes the comparison to fail. In the future we will probably need to move away from arrays as may hurt our performance anyway

Reply:
There is no `Assert.SequenceEqual`. `Assert.Equal` supports lists so changed to that